File: C99/1000-1099/1040-1049/1043.c
#include <stdio.h>

int main()
{
    double n1, n2, n3, valor;
    scanf("%lf %lf %lf", &n1, &n2, &n3);
    if (n1 + n2 > n3 && n1 + n3 > n2 && n2 + n3 > n1)
    {
        valor = n1 + n2 + n3;
        printf("Perimetro = %.1lf\n", valor);
    }
    else
    {
        valor = ((n1 + n2) * n3) / 2;
        printf("Area = %.1lf\n", valor);
    }
    return 0;
}

File: beecrowd/1000-1099/1040-1049/1043.py
entry = input().split()
entry = [eval(i) for i in entry]
if entry[0] + entry[1] > entry[2] and entry[0] + entry[2] > entry[1] and entry[1] + entry[2] > entry[0]:
    print(f"Perimetro = {sum(entry):.1f}")
else:
    print(f"Area = {((entry[0] + entry[1]) * entry[2]) / 2:.1f}")